azure-stack/hci/concepts/azure-stack-hci-dashboard.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Azure Stack HCI dashboard
+description: Learn to view and navigate your Azure Stack HCI dashboard in Azure portal.
+ms.topic: conceptual
```

Comment:
Monitor at scale Azure Stack HCI systems using the overview page in the portal (preview)

Reply:
Done > updated.